### PR TITLE
TK-211732. Stops event propagation.

### PR DESCRIPTION
**Pull request by crowtherdr**
At least in the case of the dismiss suggestion reason dialog, this prevents the close when clicking Cancel on it from also closing the indicator flyout.

@skye2k2 reviewed this with me and was ok with the change.

## Changes
- Added evt.stopPropogation

### CODE REVIEW

File: fs-common-dialog.html [hunk 1]
```diff
@@ -344,7 +344,8 @@ <h4 class="fs-h4" data-test="title" tabindex="0">[[titleText]]</h4>
        * Ensure that promptTextareaValue and errorMessage are reset and the dialog is closed after running the user-provided function (if there is one).
        * @returns {undefined} - Calls userCancelFunction.
        */
-      _cancelFunction: function(){
+      _cancelFunction: function(evt){
```
Comment (Norman-Amy) [inline fs-common-dialog.html]: There needs to be a jsdocs update for the new parameter.

Comment (crowtherdr) [inline fs-common-dialog.html]: Thanks!

**Comment by crowtherdr** [comment]
I can't tell which line Code Climate is complaining abt when it complains about "Description must start with an uppercase character." and it refers to "/**"

**Comment by Norman-Amy** [comment]
@crowtherdr CodeClimate is referring to the description of the parameter (after the dash). Change it from "dialog-closed Event" to "Dialog-closed Event".

**Comment by crowtherdr** [comment]
Thanks

**Comment by crowtherdr** [comment]
Ok. Now it's ready.

**Comment by crowtherdr** [comment]
What if Code Climate & TravisCI are not really automated processes, but controlled by people looking at our code and making up random errors and then giggling with each other?

**Comment by crowtherdr** [comment]
Thanks

**Pull request merged**